python-sdk/example_dags/example_load_file.py

```diff
@@ -1,3 +1,17 @@
+"""
+Pre-requisites for load_file_example_19:
+ - Install dependencies for Astro Python SDK with Google, refer to README.md
+ - You can either specify a service account key file and set `GOOGLE_APPLICATION_CREDENTIALS`
+    with the file path to the service account.
+ - In the connection we need to specfiy the scopes.
+    For ex:- https://www.googleapis.com/auth/drive.readonly
+    Please refer to https://developers.google.com/identity/protocols/oauth2/scopes#drive for more details.
+ - In the service account the Google Drive API must be enabled.
```

Comment:
the Google Drive API must be enabled at project level (in Google cloud), not for service account

Reply:
Modified